projects/bms_carrier/inc/fan_control.h

```diff
@@ -2,18 +2,34 @@
 
 #include <stdint.h>
 
+#include "adt7476a_fan_controller.h"
+#include "adt7476a_fan_controller_defs.h"
 #include "i2c.h"
+#include "soft_timer.h"
 #include "status.h"
 
 #include "cell_sense.h"
 
-#define BMS_FAN_CTRL_I2C_PORT_1 TBD
+#define BMS_FAN_CTRL_I2C_PORT_1 BMS_PERIPH_I2C_PORT
 #define BMS_FAN_CTRL_I2C_PORT_2 TBD
-#define NUM_BMS_FANS 4
+#define NUM_BMS_PORTS 2
+#define ADT_7476A_INTERRUPT_MASK_OFFSET 2
+#define FAN_TEMP_POLL_INTERVAL_US 500000
```

Comment:
just by convention we tend to use milliseconds rather than microseconds, tends to be easier to read (along with `soft_timer_start_millis()`)